app/src/main/java/com/github/swent/echo/compose/association/AssociationSearchBottomSheet.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.swent.echo.compose.association
+
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.runtime.Composable
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AssociationSearchBottomSheet(
+    onFullyExtended: () -> Unit,
+    onDismiss: () -> Unit,
+    searchEntryCallback: (String) -> Unit,
+    searched: String
+) {}
```

Comment:
another placeholder, seems like a lot of work to me for other branches, maybe you could request help for all the work.

Reply:
I already finished everything on an intermediate PR ! Just pushing one by one and adding the tests ;)